integration-tests/test/fee-payment.spec.ts

```diff
@@ -36,11 +37,11 @@ describe('Fee Payment Integration Tests', async () => {
   it('Should estimateGas with recoverable L2 gasLimit', async () => {
     const gas = await env.ovmEth.estimateGas.transfer(
```

Comment:
Should a `provider.getBalance` be used to populate this number?

Reply:
Meh maybe but seems overkill since it's easy to fund an account on L2 with this much eth